src/steps/create_pull_request_step.go

```diff
@@ -10,12 +10,12 @@ import (
 type CreatePullRequestStep struct {
 	NoOpStep
 	BranchName string
+	Driver     drivers.CodeHostingDriver
 }
 
 // Run executes this step.
 func (step *CreatePullRequestStep) Run(repo *git.ProdRepo) error {
-	driver := drivers.GetActiveDriver()
 	parentBranch := repo.GetParentBranch(step.BranchName)
-	script.OpenBrowser(driver.GetNewPullRequestURL(step.BranchName, parentBranch))
+	script.OpenBrowser(step.Driver.GetNewPullRequestURL(step.BranchName, parentBranch))
```

Comment:
Shouldn't a non/implemented error be cached here to give meaningful user feedback in the step's context?